pyccel/codegen/printing/ccode.py

```diff
@@ -573,11 +571,8 @@ def function_signature(self, expr):
         if len(expr.results) == 1:
             ret_type = self.get_declare_type(expr.results[0])
         elif len(expr.results) > 1:
-            ret_type = self._print(datatype('bool')) + ' '
-            if not self._additional_args :
-                self._additional_args = [a.clone(name = a.name, is_pointer =True) for a in expr.results]
-            args += self._additional_args
-            self._additional_args.clear()
+            ret_type = self._print(datatype('int')) + ' '
```

Comment:
Why did you change the return type back?

Reply:
I changed to int return 0 for no error because If a function with multiple results  always true I don't see were it can help it's like writing if True

Reply:
Ah ok, I didn't spot the return 0 :+1: